app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ReviewRegistrationController.scala

```diff
@@ -62,21 +62,8 @@ class ReviewRegistrationController @Inject() (
 
   def displayPage(): Action[AnyContent] =
     (authenticate andThen journeyAction).async { implicit request =>
-      if (request.registration.isCheckAndSubmitReady) {
-        val reg: Registration = removePartialGroupMembers(request.registration)
-
-        markRegistrationAsReviewed(reg).map(
-          _ =>
-            if (reg.isFirstGroupMember)
-              Redirect(routes.TaskListController.displayPage())
-            else
-              Ok(
-                reviewRegistrationPage(registration = reg,
-                                       liabilityStartLink = startRegistrationController.startLink
-                )
-              )
-        )
-      } else
+      if (request.registration.isCheckAndSubmitReady) reviewRegistration
```

Comment:
I think it looks a little weird the function call without the brackets: ().

Reply:
added ().